browser/src/client_win.cc

```diff
@@ -28,15 +28,20 @@ ClientWin::ClientWin(Config config, int* rc) : ClientBase(std::move(config)) {
 
   std::string pipename =
     internal::GetPipeName(configuration().name, configuration().user_specific);
-  if (pipename.empty()) {
-    return;
+  if (!pipename.empty()) {
+    unsigned long pid;
```

Comment:
nit: 0 initialize this just in case the code changes to avoid uninitialized memory issues.